client/src/components/CreatePost.test.tsx

```diff
@@ -111,16 +111,13 @@ describe('verifying launch of create post component', () => {
     expect(tagTextField.disabled).toBeTruthy();
     expect(screen.getByText('tag1')).toBeInTheDocument();
     expect(screen.getByText('tag2')).toBeInTheDocument();
-    expect(screen.getByText('tag3')).toBeInTheDocument();
-
-    fireEvent.keyDown(tagTextField, {
-      key: 'Backspace',
-    });
+    expect(screen.getByText('tag2')).toBeInTheDocument();
+    fireEvent.click(screen.getByTestId('tag2-clicker'));
```

Comment:
tag2-clicker?

Reply:
The tags have an 'x' button on them which you can click which deletes the tag. Backspacing no longer deletes and puts the tag back in the input box instead.

The testid is for the x button inside Tag.tsx